modules/smart-agent_systemd-timers/conf/02-timer-has-failed.yaml

```
@@ -0,0 +1,13 @@
+module: systemd
+name: "timer has failed"
```

```suggestion
name: "last execution state"
```

with a rule description set to `is failed` ?